builtin/game/fine_pointed.lua

```diff
@@ -0,0 +1,35 @@
+-- Returns the exact coordinate of a pointed surface
+
+local function fine_pointed(pos, camera_pos, pos_off, look_dir)
+	local offset, nc
+	local oc = {}
+	for c, v in pairs(pos_off) do
+		if v == 0 then
+			oc[#oc + 1] = c
+		else
+			offset = v
+			nc = c
+		end
+	end
+
+	local fine_pos = {[nc] = pos[nc] + offset}
+
+	camera_pos.y = camera_pos.y + 1.625
```

Comment:
To make this function usable for various height players how about removing this line and passing 'player_pos + 1.625' to this function? then the 'camera_pos' argument is correctly named.